pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go

```diff
@@ -224,6 +224,20 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 		return fmt.Errorf("create PXC Service: %v", err)
 	}
 
+	// PodDistributedBudget object for nodes
+	if cr.Spec.PXC.PodDisruptionBudget != nil {
+		pdb := pxc.NewPodDistributedBudget(cr, cr.Spec.PXC.PodDisruptionBudget, "-nodes")
+		err = setControllerReference(cr, pdb, r.scheme)
```

Comment:
I believe `PodDisruptionBudget` should be referenced to the `StatefulSet` not `CR`.
_the same with the Proxy's one_

Reply:
Ok. Thank you for your suggestion.